dxr/plugins/python/indexers.py

```diff
@@ -197,6 +204,13 @@ def visitor(self):
             self._visitor.visit(syntax_tree)
         return self._visitor
 
+    def needles(self):
+        # Index module name. For practical purposes, this includes
+        # __init__.py files for packages even though that's not
+        # _technically_ a module.
+        yield file_needle('py_module', name=local_name(self.module_name),
```

Comment:
The style I've been trying to follow is "either all args on 1 line (if they fit and read easily), or 1 arg per line".